source/tutorials/kata.rst

```diff
@@ -37,89 +37,96 @@ Install Kata Containers
 ***********************
 
 Kata Containers is included in the :file:`containers-virt` bundle.
-To install the framework, enter the following command:
+To install the framework:
 
-.. code-block:: bash
+#. Install the containers-virt bundle:
 
-   sudo swupd bundle-add containers-virt
+   .. code-block:: bash
 
-Restart the Docker\* and Kata Containers systemd services.
+      sudo swupd bundle-add containers-virt
 
-.. code-block:: bash
+#. Reload and restart the Docker\* systemd service.
 
-   sudo systemctl daemon-reload
-   sudo systemctl restart docker
+   .. code-block:: bash
+
+      sudo systemctl daemon-reload
+      sudo systemctl restart docker
 
 Run Kata Containers
 *******************
 
+To use kata as the runtime for an individual container, add
+:command:`--runtime=kata-runtime` to the :command:`docker run` command. For
+example:
+
 .. code-block:: bash
 
-   sudo docker run -ti busybox sh
+   sudo docker run --runtime=kata-runtime -ti busybox sh
+
+
+To use kata as the default runtime for all Docker containers:
 
-.. note::
+#. Set the default runtime for the Docker daemon:
 
-   If you use a proxy server and your proxy environment variables are already
-   set, run the following commands as a shell script to configure Docker:
+   .. note:: 
+
+      The method below uses a systemd drop-in configuration to add a
+      command-line (CLI) parameter to the Docker daemon for setting the
+      default runtime. Alternatively, the default runtime can be set in the
+      `Docker daemon configuration file
+      <https://docs.docker.com/engine/reference/commandline/dockerd/#daemon-configuration-file>`_.
```

Comment:
I am good with this but if you wanted to push one more commit it would be helpful to mention that if settings are set in two different places that conflict then Docker daemon will fail to start.

Reply:
Otherwise I approve it as it is.

Reply:
Good point. I've added a sentence calling it out.